Cargo.toml

```diff
@@ -39,7 +39,9 @@ required-features = ["server"]
 
 [dependencies]
 # Farcaster crates
-farcaster_core = { version = "0.4.3", features = ["serde"] }
+# farcaster_core = { version = "0.4.4", features = ["serde"] }
+farcaster_core = { git = "https://github.com/farcaster-project/farcaster-core", branch = "main", features = ["serde"] }
```

Comment:
I just did a release of core so that we don't have this wart anymore, but it failed since we currently have our own fork of secp256kfun in there. But I'm good to merge without this fixed.